pom.xml

```diff
@@ -79,10 +79,15 @@
             <dependency>
                 <groupId>com.github.cafdataprocessing</groupId>
                 <artifactId>worker-document-framework</artifactId>
-                <version>4.5.0-820</version>
+                <version>5.1.0-1038</version>
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
+            <dependency>
+                <groupId>com.github.workerframework</groupId>
+                <artifactId>standard-worker-container</artifactId>
+                <version>5.1.0-919</version>
+            </dependency>
```

Comment:
I don't know why this is being added but I'm sure there's a good reason.

Reply:
the workerframework is brought by the worker-document-framework here which brings the latest version. I only listed here as it's a transient dependency, but I agree it can be argued that it shouldn't be part of the case, I ll remove the reference from the pr and merge if that's fine

Reply:
Ok